### ISSUE TITLE
Accessibility (WCAG level AA) Select element text on the Packages page have insufficient color contrast against their backgrounds

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1169]
1.  Download Paciello Group's Contrast Analyser from http://www.paciellogroup.com/resources/contrastAnalyser.  This tool measures foreground-to-background contrast ratios using the same algorithm defined by the W3C's Web Content Accessibility Guidelines.
2.  Navigate to https://nuget.org/packages.  
3.  Compare the contrast ratio of the text in the two select elements (either stable/prerelease or 'sort by') on the page against their backgrounds.

Result:  It's 3.53:1.  
Expected:  This is below the threshold of 4.5:1 given in WCAG.  We should be able to fix this easily by slightly adjusting our color palette.

**Comment by johnataylor** [comment]
We have a new gallery design.
